package.json

```diff
@@ -39,13 +39,13 @@
         "e2etst": "npm run "
     },
     "dependencies": {
-        "@azure/functions": "^1.2.3",
+        "@azure/functions": "^3.5.0-alpha.5",

```

Comment:
I don't wanna release durable referencing an alpha version here. My plan was to release the real 3.5.0 package as soon as host 4.14 finished as that is the first version supporting our new npm package pattern. 4.14 is hopefully finishing soon, but let me know if you want the real 3.5.0 out sooner and we can probably do that for you

Reply:
I think that's fine. This fix probably won't make a difference until the new flag rolls out in Azure anyway